modules/mig/variables.tf

```diff
@@ -128,19 +128,19 @@ variable "cooldown_period" {
 
 variable "autoscaling_cpu" {
   description = "Autoscaling, cpu utilization policy block as single element array. https://www.terraform.io/docs/providers/google/r/compute_autoscaler.html#cpu_utilization"
-  type        = "list"
+  type        = list(map(number))
   default     = []
 }
 
 variable "autoscaling_metric" {
   description = "Autoscaling, metric policy block as single element array. https://www.terraform.io/docs/providers/google/r/compute_autoscaler.html#metric"
-  type        = "list"
+  type        = list(map(string))
   default     = []
 }
 
 variable "autoscaling_lb" {
   description = "Autoscaling, load balancing utilization policy block as single element array. https://www.terraform.io/docs/providers/google/r/compute_autoscaler.html#load_balancing_utilization"
-  type        = "list"
+  type        = list(map(string))
```

Comment:
This should be typed as `list(map(number))`.

Reply:
I'll fix it